CMakeLists.txt

```diff
@@ -243,7 +243,11 @@ if( DEFINED CK_OVERRIDE_HIP_VERSION_PATCH )
 endif()
 message(STATUS "Build with HIP ${HIP_VERSION}")
 link_libraries(hip::device)
-add_compile_definitions(__HIP_PLATFORM_HCC__=1)
+if(CK_hip_VERSION VERSION_GREATER_EQUAL 6.0.23494)
+    add_compile_definitions(__HIP_PLATFORM_AMD__=1)
+else()
+    add_compile_definitions(__HIP_PLATFORM_HCC__=1)
+endif()
```

Comment:
@illsilin @junliume I think that offline compiler must add these definitions on its own, these are predefined macros. In MIOpen we have a patch for HIPRTC because at least at the beginning HIPRTC path didn't provide those. I recommend removing this.

Reply:
@illsilin could we remove them? Since CK uses offline compiler at the moment. Or this is noop and will not have any effects.

Reply:
I don't think we use those macros in CK, so it makes no difference.